cmd/swarmctl/secrets/create.go

```diff
@@ -0,0 +1,44 @@
+package secrets
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/docker/swarmkit/api"
+	"github.com/docker/swarmkit/cmd/swarmctl/common"
+	"github.com/spf13/cobra"
+)
+
+var createCmd = &cobra.Command{
+	Use:   "create <secret name>",
+	Short: "Create a secret",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
+			return errors.New("create command takes a unique secret name as an argument, and accepts secret data via stdin")
```

Comment:
Oh wait, sorry, I misread the first comment.  I think taking a secret as an argument may not be great, since it's visible on the shell history.  I was +1 on passing in a file via `-f` argument, but I don't feel super strongly between that and @aaronlehmann's stdin.  The positive thing about @aaronlehmann's version of stdin being constructed by pipes was being able to pipe it in from some other source (e.g. maybe some other command line utility)